bin/dev/make_credits.py

```diff
@@ -101,7 +101,7 @@
 print('> found a total of %d contributors and translators' % n3)
 contributors.sort(key=str.lower)
 
-f = open(path+'/util/credits.txt', 'w')
+f = open(path+'/CREDITS.txt', 'w')
```

Comment:
I don't think that the base directory is a good place for this file as it is not needed for having Friendica running. The `util` directory was a good place for it as it a utility file. For me it does not fit into `doc` either.

Reply:
For me it fits with LICENCE and VERSION. However `database.sql` and `friendica_test_data.sql` shouldn't be in the base directory but I haven't found a satisfactory directory to move them to.